### PR TITLE
Create HGLR-HGLRC_F722_MINI_SP.config

### PR DESCRIPTION
**Pull request by HGLRC-T**
Create HGLR-HGLRC_F722_MINI_SP (include hw define)

### CODE REVIEW

File: configs/default/HGLR-HGLRC_F722_MINI_SP.config [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACCGYRO_BMI270
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_MAX7456
+#define USE_FLASH_W25Q128FV
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
```
Comment (haslinghuis) [inline configs/default/HGLR-HGLRC_F722_MINI_SP.config:11]:
```suggestion
#define USE_ACC
#define USE_ACC_SPI_MPU6000
#define USE_ACC_SPI_ICM42688P
#define USE_ACCGYRO_BMI270
#define USE_BARO
#define USE_BARO_BMP280
#define USE_BARO_DPS310
#define USE_FLASH
#define USE_FLASH_W25Q128FV
#define USE_GYRO
#define USE_GYRO_SPI_MPU6000
#define USE_GYRO_SPI_ICM42688P
#define USE_MAX7456
```

File: configs/default/HGLR-HGLRC_F722_MINI_SP.config [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set blackbox_device = SPIFLASH
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 250
+set beeper_inversion = ON
+set beeper_od = OFF
+set max7456_spi_bus = 2
+set flash_spi_bus = 2
```
Comment (haslinghuis) [inline configs/default/HGLR-HGLRC_F722_MINI_SP.config:116]: SPI sharing is not recommended and unsupported.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/HGLR-HGLRC_F722_MINI_SP.config (hunk 2 above)
Comment (HGLRC-T) [reply configs/default/HGLR-HGLRC_F722_MINI_SP.config:120]: > SPI sharing is not recommended and unsupported.
We can identify and use it in actual testing. Thank you very much for your suggestion

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — CHANGES_REQUESTED** [review]
PR is based on master - not feature branch

**Pull request closed without merging**